### DISCUSSION TITLE
Making basic storyboard sounds available to be able to play o2jam converts.

### DISCUSSION THREAD

**Discussion by dtrdpixl** [ppy/osu#29750] [Q&A] [answered]
Greetings, I've been playing mania in lazer for a while, and noticed that most o2jam convert packs aren't able to be played due to the songs being added as storyboard sounds (due to how the converter works) and wanted to know if it's possible to add very basic storyboard sample functionality in order for the songs to be played in game.

Thanks in advance for seeing this message!

(Image about how the songs are added to o2jam converts in osu!)
![imagen](https://github.com/user-attachments/assets/331eb02a-2bf9-4c60-b0e2-f5fa5199c261)

**Comment by peppy** [comment]
This should be implemented. DO you have storyboards turned off or something?

**Reply by dtrdpixl** [reply]
Yes, HItsounds and ranked storyboarded maps with keysounds do work, this problem only and specifically occurs with converted o2jam maps. 

In case you want to replicate it, here is one of those packs:
[https://onedrive.live.com/?authkey=%21AC%2DLvtjSYeixc1o&cid=4CF08022FEB1D9AB&sb=size&sd=1&id=4CF08022FEB1D9AB%2111069&parId=4CF08022FEB1D9AB%2111070&o=OneUp](url)

you can drop any of those songs into stable and they will work, but once opened in lazer, there will only be keysounds, but no music.

**Reply by peppy** [reply]
Thanks, will take a look.

**Reply by peppy** [reply]
They seem to work for me. Please make sure you have the "Storyboard / video" toggle on.

**Reply by dtrdpixl** [reply]
I'm pretty sure I tried toggling the ''Storyboard / video'' before

This is what happens on my end:
https://www.youtube.com/watch?v=hnmA7jPDwig

**Reply by dtrdpixl** [reply, marked as answer]
It works after the latest update WWW!
